nwb_conversion_tools/gui/classes/forms_basic.py

```diff
@@ -105,11 +105,20 @@ def make_forms(self):
             # String types
             if field['type'] == 'str':
                 form = QLineEdit('')
-                form.setText(field['default'])
+                if field['required']:
+                    form.setPlaceholderText(field['default'])
+                else:
+                    form.setText(field['default'])
             # Float types
             elif field['type'] == 'float':
-                form = QLineEdit('')
-                form.setText(str(field['default']))
+                if field['default'] is not None:
+                    form = QLineEdit('')
+                    if field['required']:
+                        form.setPlaceholderText(str(field['default']))
```

Comment:
A default value should fill whether or not it is required. This applies throughout.